src/buildercore/cfngen.py

```diff
@@ -350,34 +372,46 @@ def generate_stack(pname, **more_context):
     """given a project name and any context overrides, generates a Cloudformation
     stack file, writes it to file and returns a pair of (context, stackfilename)"""
     context = build_context(pname, **more_context)
-    template = render_template(context)
+    cloudformation_template = render_template(context)
```

Comment:
can see the asymmetry here: should be `cloudformation_template = cloudformation.render(context)`. I plan to extract a `cloudformation` module from this, as `cfngen` has grown with more than just those specific concerns. This should become a `context generator` rather than a `cloudformation generator` if that was the original meaning (but I am *very* hesitant to rename modules, have to live with it for a while).

Reply:
it's fine. `render_template` is just calling trop.py, so we could rename trop.py to cloudformation.py 

if you look at `render_template` it looks like you or I built in support for other template types in the future, but we could simply shift that function into the new cloudformation.py module and have a matching one in terraform.py. or use `render_template` to dispatch calls to the other modules, I don't mind.